currencies/src/lib.rs

```diff
@@ -156,6 +157,24 @@ pub mod module {
 			Ok(().into())
 		}
 
+		/// Transfer all remaining balance to the given account under `currency_id`.
+		///
+		/// The dispatch origin for this call must be `Signed` by the
+		/// transactor.
+		#[pallet::weight(T::WeightInfo::transfer_all_non_native_currency())]
+		pub fn transfer_all(
+			origin: OriginFor<T>,
+			dest: <T::Lookup as StaticLookup>::Source,
+			currency_id: CurrencyIdOf<T>,
+		) -> DispatchResultWithPostInfo {
+			let from = ensure_signed(origin)?;
+			let to = T::Lookup::lookup(dest)?;
+			let balance = <Self as MultiCurrency<T::AccountId>>::free_balance(currency_id, &from);
```

Comment:
this needs to be free - frozen

Reply:
Do you mean apply this for `transfer_all` in token module as we only define it in `orml-tokens`